set-versions

```diff
@@ -38,7 +38,7 @@ for project in $(find . -name project.clj -exec dirname '{}' \;); do
 
   cljproj=$(basename $project)
 
-  if [ "$cljproj" == "Donkey" ]; then
+  if [ "$cljproj" == "Terrain" ]; then
```

Comment:
@johnworth will probably be the one to ask

And as far as "out of scope" is concerned, they're not out of scope comments.
As long as any other issues discovered are written up, it is all part of the review process